.ci/local_integration_test

```diff
@@ -31,6 +64,140 @@ if ! hash ginkgo; then
     echo "Successfully fetched Ginkgo frawework"
 fi
 
-echo "Starting integration tests..."
+printf  "\e[34mIs the control cluster a seed ? (true/false)\e[0m\n"
+
+read IS_CONTROL_CLUSTER_SEED
 
+if [ -z "$IS_CONTROL_CLUSTER_SEED" ]
+then
+      printf "\e[31mIS_CONTROL_CLUSTER_SEED cannot be empty\e[0m\n"
+      exit 1
+fi
+
+if [ "$IS_CONTROL_CLUSTER_SEED" != true ] && [ "$IS_CONTROL_CLUSTER_SEED" != false ]
+then
+      printf "\e[31mIS_CONTROL_CLUSTER_SEED must be a boolean\e[0m\n"
+      exit 1
+fi
+export IS_CONTROL_CLUSTER_SEED=$IS_CONTROL_CLUSTER_SEED
+
+printf "\e[34mEnter the absolute path to the local machine-controller-manager repository.\e[0m\n"
+
+read MCM_REPO_PATH
+if [ -z "$MCM_REPO_PATH" ]
+then
+      printf "\e[31mMCM_REPO_PATH cannot be empty\e[0m\n"
+      exit 1
+fi
+if [ ! -d "$MCM_REPO_PATH" ]
+then
+      printf "\e[31m$MCM_REPO_PATH is not a valid directory\e[0m\n"
+      exit 1
+fi
+
+if [ "$IS_CONTROL_CLUSTER_SEED" = true ]
+then
+    cd $MCM_REPO_PATH
+    make gardener-setup
+    cd -
+else
+    cd $MCM_REPO_PATH
+    echo "Enter the project"
+    read PROJECT
+    echo "Enter the control cluster"
+    read CONTROL_CLUSTER
+    echo "Enter the target cluster"
+    read TARGET_CLUSTER
+    echo "Enter the machine class path"
```

Comment:
Should we have validations for these entered values?
maybe basic ones, like check for empty value